select_ppx_version.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh -e
 
+case $1 in
+  clean) rm -f ppx/ppx_cstruct.ml; exit 0 ;;
+  conf) (* continue *) ;;
```

Comment:
looking at the patch again, this line uses OCaml comment syntax, which provokes the following behaviour from the root of the repository:

```$ (* continue *)
bash: appveyor.yml: command not found
```

(The first `*` expands to the list of files in alphabetic order and the first, `appveyor.ml`, gets called as a command)

It would be nice to commit a minor change to remove the comment or use proper shell comment syntax.